### PR TITLE
Add integrity classname

### PR DESCRIPTION
**Pull request by damassi**
We're using a new `<MinimalNavBar>` component and it needs an identifier for our acceptance tests.

### CODE REVIEW

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.11.1` :rocket:

File: src/Components/NavBar/MinimalNavBar.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const MinimalNavBar: React.FC<MinimalNavBarProps> = props => {
     >
       <AppContainer>
         <Box height={70} px={[2, 4]}>
-          <RouterLink to={props.to}>
+          <RouterLink to={props.to} className="acceptance__logoLink">
```
Comment (sweir27) [inline src/Components/NavBar/MinimalNavBar.tsx]: @damassi can you please use the `data-test` attribute here as we do in [other places](https://github.com/artsy/reaction/pull/3089)?

It's a small change but best to be consistent!

Comment (damassi) [reply src/Components/NavBar/MinimalNavBar.tsx]: Oh this is already in :)

Comment (sweir27) [reply src/Components/NavBar/MinimalNavBar.tsx]: I know! I am suggesting a quick update.

The integrity tests are primarily failing for another reason (an inability to access cms-staging), so there's not a huge rush over there. 😄

Comment (damassi) [reply src/Components/NavBar/MinimalNavBar.tsx]: (No meaning, i've already updated reaction with `data-test`)

Comment (sweir27) [reply src/Components/NavBar/MinimalNavBar.tsx]: Oooh 👍

**Pull request merged**